docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -46,10 +49,104 @@ Data Description
 
    * - State
      - Definition
+   * - S
+     - Susceptible but does not currently have LRI
    * - I
      - Currently infected and having the condition
+
+.. list-table:: **States Data**
+   :widths: 20 25 30 30
+   :header-rows: 1
+
+   * - State
+     - Measure
+     - Value
+     - Notes
    * - S
-     - Susceptible but does not currently have LRI
+     - prevalence
+     - 1-prevalence_c322
+     -
+   * - S
+     - excess mortality |br| rate
```

Comment:
these |br| comments are showing up strangely in the compiled version. maybe remove?